src/Microsoft.ML.Data/Prediction/IPredictionTransformer.cs

```diff
@@ -15,7 +15,6 @@ namespace Microsoft.ML.Runtime
     /// </summary>
     /// <typeparam name="TModel">The <see cref="IPredictor"/> used for the data transformation.</typeparam>
     public interface IPredictionTransformer<out TModel> : ITransformer
-        where TModel : IPredictor
     {
```

Comment:
Why did this get removed? Do you need to modify the doc string to reflect this change?

Reply:
got removed because the calibrator transformers derive from here, and their TModel is an ICalibrator. 

---
In reply to: [240343227](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240343227) [](ancestors = 240343227)